bin/update-theos

```diff
@@ -5,22 +5,30 @@ has() {
 	type "$1" >/dev/null 2>&1
 }
 
+# Root is no bueno
+if [[ $EUID -eq 0 ]]; then
+	echo "Theos should NOT be installed with or run as root (su/sudo)!" >&2
+	echo "  - If you've installed Theos as root, please remove the install and reinstall as a non-root user." >&2
+	echo "  - If you're using root because you don't have write permission for Theos' parent directory, please either install Theos to a different directory where you do have write permission or, if you're trying to install to /opt, change the directory's ownership before attempting to install Theos as a non-root user." >&2
```

Comment:
This text looks good. Could you wrap it at 80 columns? Good to merge once that's done.